### PR TITLE
Fix aarch64 linux CI. Add FreeBSD, M1

### PR DESCRIPTION
**Pull request by IanButterworth**
Skipping musl because of https://github.com/JuliaLang/julia/issues/40556
```
 Running tests...
ERROR: 
LoadError: InitError: could not load library "/root/.julia/artifacts/ee46ae25e760393f5771967db1bef786291ac497/lib/libfreetype.so"
Error loading shared library libbz2.so.1.0: No such file or directory (needed by /root/.julia/artifacts/ee46ae25e760393f5771967db1bef786291ac497/lib/libfreetype.so)
```

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        sh -c "$(fetch ${URL} -o -)"
+    else
+        echo "No downloader installed! :( Install one! :)"
+    fi
+  build_script:
+    - cirrusjl build
+  test_script:
+    - cirrusjl test
+  coverage_script:
+    - cirrusjl coverage codecov
```
Comment (giordano) [inline .cirrus.yml:42]: Missing trailing newline
```suggestion
    - cirrusjl coverage codecov

```

**Comment by IanButterworth** [comment]
Skipping musl because of https://github.com/JuliaLang/julia/issues/40556

File: .cirrus.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        image: ubuntu:latest
+    # - name: musl Linux # skip because of https://github.com/JuliaLang/julia/issues/40556
+    #   container:
+    #     image: alpine:3.14
+    - name: MacOS M1
+      macos_instance:
+        image: ghcr.io/cirruslabs/macos-monterey-base:latest
+  env:
+    matrix:
+      - JULIA_VERSION: 1.6
```
Comment (giordano) [inline .cirrus.yml:17]: There is no 1.6 for M1

File: test/runtests.jl [hunk 3]
```diff
@@ -1,3 +1,5 @@
+haskey(ENV, "LD_LIBRARY_PATH") && @show ENV["LD_LIBRARY_PATH"]
+
```
Comment (giordano) [inline test/runtests.jl:2]: Not really needed anymore.

**Comment by IanButterworth** [comment]
Appears to be being OOM-killed on ARMv8

@giordano I guess that means it should be skipped, unless you have any suggestions?

**Comment by giordano** [comment]
Sigh, yes

**Pull request merged**